src/transformers/models/jetmoe/modeling_jetmoe.py

```diff
@@ -1343,7 +1343,7 @@ def prepare_inputs_for_generation(
     """,
     JETMOE_START_DOCSTRING,
 )
-# Copied from transformers.models.llama.modeling_llama.LlamaForSequenceClassification with Llama->JetMoe, LLAMA->JETMOE
+# Copied from transformers.models.mistral.modeling_mistral.MistralForSequenceClassification with Mistral->JetMoe, MISTRAL->JETMOE
```

Comment:
instead of adding a todo on all models, `ForSequenceClassification` and `ForTokenClassification` are using `mistral` as a pivot :)